src/core/AutoRest/Program.cs

```diff
@@ -91,13 +91,6 @@ private static int Main(string[] args)
                             return 0;
                         }
 
-                        // comparison?
```

Comment:
Quick Question -- do we even support an execution path that runs `main() ` without `--server` (maybe for `-help`, which we should remove too.)